lighthouse-core/report/html/renderer/util.js

```diff
@@ -477,6 +477,8 @@ Util.UIStrings = {
   warningHeader: 'Warnings: ',
   /** The tooltip text on an expandable chevron icon. Clicking the icon expands a section to reveal a list of audit results that was hidden by default. */
   auditGroupExpandTooltip: 'Show audits',
+  /** Section heading shown above a list of audits that contain warnings. */
+  warningAuditsGroupTitle: 'Audits with warnings',
```

Comment:
I feel like we should be more specific with something like `Passed audits with warnings`. And is "with warnings" how we want to phrase it?

Reply:
wb `Passed but contains (or with) warnings`? the manual and the n/a titles don't explicitly say "audit".